lib/mongo/collection/view/readable.rb

```diff
@@ -281,6 +289,7 @@ def estimated_document_count(opts = {})
                   db_name: database.name,
                   read: read_pref,
                   session: session,
+                  comment: opts[comment],
```

Comment:
I was looking at the spec for `estimatedDocumentCount` and it doesn't look like the comment field is included there. Is that a mistake here or there? Or I could just be reading the spec wrong.

Reply:
Or are all the options from CountOptions included in EstimatedCountOptions as well?

Reply:
I think I just forgot to add `comment` to `EstimatedCountOptions`. This helper actually uses `count` command, so `comment` works.
I'll create a DRIVERS ticket and PR to fix `EstimatedCountOptions`. I think we do not need to wait until spec is updated, and can leave this change.